docs/Bots.md

```diff
@@ -50,8 +344,207 @@ The configuration is called `modify.conf` and looks like this:
 
 Assume we have an event with `feed.name = Spamhaus Cert` and `malware.name = feodo`. The default condition matches, but no others. So the default action is applied. The value for `classification.identifier` is `{msg[malware.name]}`, this is [standard Python string format syntax](https://docs.python.org/3/library/string.html#formatspec). Thus you can use any value from the processed event, which is available as `msg`.
+
+
+
+
+<a name="outputs"></a>
+## Outputs
+
+#### File
+
+**Information:**
+* `name:` file
+* `lookup:` no
+* `public:` yes
+* `cache (redis db):` none
+* `ipv6 support:` yes
+* `description:` output messages (reports or events) to file
+
+**Configuration Parameters**
+
+* `file`: filepath of output file
+
+
+* * *
+
+
+#### MongoDB
+
+**Information:**
+* `name:` mongodb
+* `lookup:` no
+* `public:` yes
+* `cache (redis db):` none
+* `ipv6 support:` yes
+* `description:` MongoDB is the bot responsible to send events to a MongoDB database
+
+**Configuration Parameters**
+
+* `collection`: MongoDB collection
+* `database`: MongoDB database
+* `host`: MongoDB host (fqdn or IP)
+* `port`: MongoDB port
+
+**Installation Requirements**
+
+Using Python 3.4 (recommended):
+```
+pip3 install pymongo>=2.7.1
+```
+
+Using Python 2.7:
+```
+pip2 install pymongo>=2.7.1
+```
+
+
+* * *
+
+
+#### IntelMQ Mailer
+
+**Information:**
+* `name:` intelmqmailer
+* `lookup:` no
+* `public:` yes
+* `cache (redis db):` none
+* `ipv6 support:` yes
+* `description:` IntelMQ Mailer is the bot responsible to send events to a MongoDB database that supports IntelMQ Mailer platform
+
+**Configuration Parameters**
+
+* `collection`: MongoDB collection
+* `database`: MongoDB database
+* `host`: MongoDB host (FQDN or IP)
+* `port`: MongoDB port
+
+**Installation Requirements**
+
+Using Python 3.4 (recommended):
+```
+pip3 install pymongo>=2.7.1
+```
+
+Using Python 2.7:
+```
+pip2 install pymongo>=2.7.1
+```
+
+
+* * *
+
+
+#### PostgreSQL
+
+**Information:**
+* `name:` postgresql
+* `lookup:` no 
+* `public:` yes
+* `cache (redis db):` none
+* `ipv6 support:` yes
+* `description:` PostgreSQL is the bot responsible to send events to a PostgreSQL Database
+* `notes`: When activating autocommit, transactions are not used: http://initd.org/psycopg/docs/connection.html#connection.autocommit
+
+**Configuration Parameters**
+
+* `autocommit`: FIXME
+* `database`: PostgreSQL database
+* `host`: PostgreSQL host
+* `port`: PostgreSQL port
+* `user`: PostgreSQL user
+* `password`: PostgreSQL password
+* `sslmode`: FIXME
+* `autocommit`: FIXME
+* `table`: FIXME
+
+**Installation Requirements**
+
+Using Python 3.4 (recommended):
+```
+pip3 install psycopg2>=2.5.5
+```
+
+Using Python 2.7:
+```
+pip2 install psycopg2>=2.5.5
+```
+
+**PostgreSQL Installation**
```

Comment:
Duplicate, as it is also in intelmq/bots/outputs/postgresql/README.md
Either remove the file or this section.